### PR TITLE
Add Zipcode Controller and Tests

### PR DESCRIPTION
**Pull request by niyati-m**
In this PR, we add an endpoint `/api/zipcode/get` that can be used to get information
about the location of a particular zipcode.

### CODE REVIEW

**Review by Jumshim — APPROVED** [review]
✅ LGTM!

**Review by vaaniarora — APPROVED** [review]
lgtm!

**Review by rainakakani21 — APPROVED** [review]
lgtm

File: src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/ZipCodeController.java [hunk 1]
```diff
@@ -1,9 +1,42 @@
...
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@Tag(name="Getting Zipcodes")
```
Comment (emilythai) [inline src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/ZipCodeController.java]: I think the name for the tag is "Zip Code Information from http://www.zippopotam.us/" seen on the staff's swagger-iu http://team01.dokku-00.cs.ucsb.edu/swagger-ui/#/zip-code-controller

**Review by charlene-patenaude — APPROVED** [review]
lgtm :)

**Review by emilythai — CHANGES_REQUESTED** [review]
I think the name for the tag is "Zip Code Information from http://www.zippopotam.us/" seen on the staff's swagger-iu http://team01.dokku-00.cs.ucsb.edu/swagger-ui/#/zip-code-controller

**Review by emilythai — APPROVED** [review]
LGTM!

**Pull request merged**